packages/server/src/migration/1630344306894-DatabaseIndexOptimization.ts

```diff
@@ -0,0 +1,39 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DatabaseIndexOptimization1630344306894 implements MigrationInterface {
+
+  name = 'DatabaseIndexOptimization1630344306894';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user_notification_setting" ALTER COLUMN "last_notified_date" SET DEFAULT null`);
```

Comment:
There's a bug in TypeORM unfortunately where it always wants to alter this column, even if it already did in the last migration. So it can safely be removed. There's a matching line in the `down` function as well that can be removed.